tasks/init_cfme.yml

```diff
@@ -0,0 +1,33 @@
+- name: Add host alias of appliance
+  no_log: true
+  add_host:
+    hostname: "{{ miq_ip_addr }}"
+    ansible_host: "{{ miq_ip_addr }}"
+    ansible_password: "{{ miq_password }}"
+    ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+
+- name: Fetch info about appliance
```

Comment:
patch validation fails for me here:
fatal: [10.12.50.53]: UNREACHABLE! => {
    "changed": false, 
    "msg": "Authentication failure.", 
    "unreachable": true
}
for some reason it's trying to connect to the engine

Reply:
It's trying to connect to '10.12.50.53', is that engine or CF?

Reply:
Most probably you've passed wrong `miq_password`